websites/H/Hyakanime/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Dem0nx309",
+		"id": "145559039173066752"
+	},
+	"service": "Hyakanime",
+	"description": {
+		"en": "French site, allows you to follow your anime progress",
+		"fr": "Site franÃ§ais, permet de suivre ta progresion anime"
```

Comment:
Taken from the website's embed, you should either use "Crée ta bibliothèque d'animés !" or "Bibliothèque et Agenda Anime" (and their corresponding translation). Also I'm pretty sure that you didn't mean to spell "franÃ§ais"